mappings/net/minecraft/entity/vehicle/VehicleEntity.mapping

```diff
@@ -2,7 +2,7 @@ CLASS net/minecraft/class_8836 net/minecraft/entity/vehicle/VehicleEntity
 	FIELD field_46655 DAMAGE_WOBBLE_TICKS Lnet/minecraft/class_2940;
 	FIELD field_46656 DAMAGE_WOBBLE_SIDE Lnet/minecraft/class_2940;
 	FIELD field_46657 DAMAGE_WOBBLE_STRENGTH Lnet/minecraft/class_2940;
-	METHOD method_54293 shouldAlwaysDropItem ()Z
+	METHOD method_54293 shouldAlwaysDestroy ()Z
```

Comment:
I disagree with this name change, though the existing name maybe isn't exactly right either?

When this returns `true` and the vehicle is damaged, if the player who hit it is in creative, the vehicle is killed (`Entity#kill`) and drops an item.
When this returns `false` and the vehicle is damaged, if the player who hit it is in creative, the vehicle is discarded (`Entity#discard`).

This value is only ever queried when the vehicle is damaged by a player in creative mode.
This value only affects whether to kill and drop item (true), or to discard (false).
No matter the value, the entity is destroyed.

Reply:
There isn't really a good way to describe the behavior, without handling the TNT minecart case. This method in particular is only overridden by TNT carts - which, you know, explode.